### PR TITLE
Add/Update some mappings for `net.minecraft.client.model.*`.

### PR DESCRIPTION
**Pull request by iChun**

### CODE REVIEW

**Comment by Lanse505** [comment]
Sorry we don't accept PRs from Ichun ;P 
(Btw sign the CLA)

**Comment by iChun** [comment]
> Sorry we don't accept PRs from Ichun ;P
> (Btw sign the CLA)

Why Simon, why?

File: data/net/minecraft/client/model/geom/ModelPart.mapping [hunk 1]
```diff
@@ -1,38 +1,87 @@
...
 		ARG 8 alpha
 	METHOD setPos (FFF)V
-		ARG 1 rotationPointX
-		ARG 2 rotationPointY
-		ARG 3 rotationPointZ
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD setRotation (FFF)V
+		ARG 1 x
```
Comment (WithoutAName25) [inline data/net/minecraft/client/model/geom/ModelPart.mapping]: Maybe take field names instead
```suggestion
		ARG 1 xRot
```

File: data/net/minecraft/client/model/geom/ModelPart.mapping [hunk 2]
```diff
@@ -1,38 +1,87 @@
...
+		ARG 2 y
```
Comment (WithoutAName25) [inline data/net/minecraft/client/model/geom/ModelPart.mapping]:
```suggestion
		ARG 2 yRot
```

File: data/net/minecraft/client/model/geom/ModelPart.mapping [hunk 3]
```diff
@@ -1,38 +1,87 @@
...
+		ARG 3 z
```
Comment (WithoutAName25) [inline data/net/minecraft/client/model/geom/ModelPart.mapping]:
```suggestion
		ARG 3 zRot
```

File: data/net/minecraft/client/model/geom/builders/CubeDefinition.mapping [hunk 4]
```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/client/model/geom/builders/CubeDefinition
+	METHOD <init> (Ljava/lang/String;FFFFFFFFLnet/minecraft/client/model/geom/builders/CubeDeformation;ZFF)V
+		ARG 1 comment
+		ARG 2 texCoordX
```
Comment (WithoutAName25) [inline data/net/minecraft/client/model/geom/builders/CubeDefinition.mapping]:
```suggestion
		ARG 2 texCoordU
```

File: data/net/minecraft/client/model/geom/builders/CubeDefinition.mapping [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+		ARG 3 texCoordY
```
Comment (WithoutAName25) [inline data/net/minecraft/client/model/geom/builders/CubeDefinition.mapping]:
```suggestion
		ARG 3 texCoordV
```

**Review by WithoutAName25 — CHANGES_REQUESTED** [review]
Some small suggestions

**Comment by iChun** [comment]
@WithoutAName25 done

**Review by WithoutAName25 — DISMISSED** [review]

File: data/net/minecraft/client/model/geom/builders/CubeDefinition.mapping [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+		ARG 1 comment
+		ARG 2 texCoordU
+		ARG 3 texCoordV
+		ARG 4 originX
+		ARG 5 originY
+		ARG 6 originZ
+		ARG 7 dimensionX
+		ARG 8 dimensionY
+		ARG 9 dimensionZ
+		ARG 10 cubeDeformation
```
Comment (sciwhiz12) [inline data/net/minecraft/client/model/geom/builders/CubeDefinition.mapping]: According to convention, this should follow the name of the field which the parameter is directly assigned to: `grow`.

**Review by sciwhiz12 — CHANGES_REQUESTED** [review]
A small thing that needs changing, otherwise it's all fine to me.

**Comment by iChun** [comment]
@sciwhiz12 Done.

Let me know if you'd like me to squash the commits as well

**Review by WithoutAName25 — APPROVED** [review]

**Review by sciwhiz12 — APPROVED** [review]
Looks good to me. No need to squash the commits on your end, standard procedure is to squash-merge PRs from GitHub.

**Pull request merged**